packages/maskbook/src/extension/options-page/DashboardComponents/CollectibleList/Card.tsx

```diff
@@ -40,7 +40,7 @@ export function CollectibleCard(props: CollectibleCardProps) {
                         height: 220,
                     }}>
                     {props.url ? (
-                        <Image component="img" width={160} height={220} src={props.url} />
+                        <Image component="img" style={{ width: '90%', height: 'auto' }} src={props.url} />
```

Comment:
fixed width and height would stretch the image inappropriately 


<img src="https://user-images.githubusercontent.com/63733714/111287727-7614ad80-867e-11eb-9078-8193f1af97e2.png" width=300 />

to

<img src="https://user-images.githubusercontent.com/63733714/111287748-7dd45200-867e-11eb-9e64-1d11574bec1c.png" width=300 />

Reply:
The `object-fit` rule is invented for this. Have it a try.

Reply:
It works, and is more readable! thanks